lib/urbanairship/configuration.rb

```diff
@@ -0,0 +1,10 @@
+module Urbanairship
+  class Configuration
+    attr_accessor :log_path, :log_level
+
+    def initialize
+      @log_path = nil
+      @log_level = Logger::DEBUG
```

Comment:
When we instantiate a logger as the gem is doing, the level is Debug.

```rb
logger = Logger.new('some.log')
logger.level
#=> 0 (Debug)
```
So, I considered to keep this behavior. And i've put here in initialize, just to not need to put a if here: https://github.com/urbanairship/ruby-library/pull/106/files#diff-24a1cfbbd59fb2f8548f1bbc519ed1d1R20

```rb
logger.level = Urbanairship.configuration.log_level if Urbanairship.configuration.log_level
```

Because, without the if, and without the default value in initialize, we would set the log level to nil :)

But if is Ok for the gem maintainers use log level info as default, for me is nice 😄